pitaya-sharp/NPitaya/src/Metrics/MetricsReporter.cs

```diff
@@ -1,103 +1,37 @@
 using System;
-using System.Runtime.InteropServices;
-using NPitaya.Models;
 
 namespace NPitaya.Metrics
 {
-    public class MetricsReporter
+    class MetricsReporter
```

Comment:
From what I understand this class is the one that instantiates both PrometheusMetrics and PitayaMetrics, how this are supposed to be used? If I need to register prometheus metrics? Why is the constructor is internal?

Reply:
Because we don't want to expose this constructor to the public, yet. Metrics instantiation are being fully controlled by the Configuration struct and their execution and iteration are being exposed by the `PitayaCluster` API. We can discuss and rearrange this kind of organization in futures PRs, but, for now, the less structures being exposed the better.

For now, the main goal of this class is instantiate all internal reporters, control the feature flag and forward metrics actions (inc, set, observe) properly.

Reply:
Ok